src/main/java/com/gluonhq/substrate/target/WindowsTargetConfiguration.java

```diff
@@ -235,7 +235,7 @@ private void clearExplorerCache() throws IOException, InterruptedException {
         ProcessRunner clearCache = new ProcessRunner("ie4uinit");
         // For Windows 10 and later, use `ie4uinit.exe -show`
         // For Windows version < 10, use `ie4uinit.exe -ClearIconCache`
-        clearCache.addArg(osName.equals("windows 10") || osName.equals("windows 11") ? "-show" : "-ClearIconCache");
+        clearCache.addArg(osName.equals("windows 10") || osName.equals("windows 11") || osName.equals("windows server 2019") ? "-show" : "-ClearIconCache");
```

Comment:
It could probably better to have a default set to `-show`, and only if windows version < 10 set the other option. The problem is finding out which versions are those (Windows 7, 8, 8.1, server versions 2016? 2012...)

Another option: use the Windows build number. Checking https://en.wikipedia.org/wiki/List_of_Microsoft_Windows_versions, it seems build numbers > 10000 could work.

Running `ver` should give you either major version and build version
```
$ ver
Microsoft Windows [Version 10.0.14366]
```